library/dialog.js

```diff
@@ -114,69 +114,117 @@ function dialogID()
   return s4() + s4() + s4() + s4() + s4() + + s4() + s4() + s4();
 }
 
-if(top.tab_mode)
-{
-    dlgOpenWindow=dlgopen;
-    dlgopen=function(url,winname,width,height,forceNewWindow)
-    {
+if (top.tab_mode) {
+    dlgOpenWindow = dlgopen;
+    dlgopen = function (url, winname, width, height, forceNewWindow, title) {
         top.restoreSession();
 
-        if(forceNewWindow)
-        {
-            return dlgOpenWindow(url,winname,width,height);
+        if (forceNewWindow) {
+            return dlgOpenWindow(url, winname, width, height);
         }
-        width=width+20;
-        height=height+20;
+
         var fullURL;
-        if(url[0]==="/")
-        {
-            fullURL=url
+        if (url[0] === "/") {
+            fullURL = url
         }
-        else
-        {
-            fullURL=window.location.href.substr(0,window.location.href.lastIndexOf("/")+1)+url;
+        else {
+            fullURL = window.location.href.substr(0, window.location.href.lastIndexOf("/") + 1) + url;
         }
-        var dialogDiv=top.$("#dialogDiv");
-        var dlgIframe={};
-        if(winname!=="_blank")
-        {
-            dlgIframe=dialogDiv.find("iframe[name='"+winname+"']");
+
+        var dialogDiv = top.$("#dialogDiv");
+        var dlgIframe = {};
+        if (winname !== "_blank") { // @TODO Check if bug below
+            // even if a frame is given this dlgIframe was over written by dlgIframe below.
+            // a bug or not needed any longer ... check for intent or where/how used.
+            dlgIframe = dialogDiv.find("iframe[name='" + winname + "']");
         }
-        else
-        {
-            winname=dialogID();
+        else {
+            winname = dialogID();
         }
 
+        // dlgIframe = top.$("<iframe></iframe>");
+        // dlgIframe.attr("name", winname);
```

Comment:
Remove commented code unless there is a good reason for it.

Reply:
Rod, I actually left those there for you to look at. I'm trying to understand why/how it came to be that the above conditional to test for an existing iframe to insert into modal could not make it past the commented code as the iframe ref was simply over written by commented code. Need to know if I need to account for an existing window or iframe...

Reply:
I didn't write that code but in the original dlgopen there was logic to check for an existing dialog window of the same name onscreen, and if found the user was asked if they wanted to re-use it. A negative response would cancel the open request. The main reason for that is users would often open a dialog without realizing it was already open (often hidden behind the main window). All that probably doesn't apply here.

Reply:
Makes sense. I'll fix and keep functionality for occasions where a normal window is needed/wanted.